packages/twind/src/internal/parse.ts

```diff
@@ -68,7 +68,7 @@ function uniq<T>(value: T, index: number, values: T[]): boolean {
 
 // Remove comments (multiline and single line)
 export function removeComments(tokens: string): string {
-  return tokens.replace(/\/\*[^]*?\*\/|\s\s+|\n/gm, ' ')
+  return tokens.replace(/\s+\/\/.*|\/\*[^]*?\*\/|\s\s+|\n/gm, ' ')
```

Comment:
Maybe we should support line comments at the start of a line?

Something like this:

```suggestion
  return tokens.replace(/(?:^|\s+)\/\/.*|\/\*[^]*?\*\/|\s\s+|\n/gm, ' ')
```